install/operator/pkg/syndesis/backup/restore.go

```diff
@@ -52,6 +65,10 @@ func (b *Backup) Restore() (err error) {
 			return err
 		}
 
+		nfi, err := os.Stat(dir)
+		if !nfi.IsDir() {
+			return fmt.Errorf("Unzipped backup directory does not exist: %s", dir)
+		}
 		b.BackupDir = dir
```

Comment:
Why is this needed, dir is surely created few lines above or?
```
		dir, err := ioutil.TempDir("/tmp", "restore-")
		if err != nil {
			return err
		}
		defer os.RemoveAll(dir)
		os.Chmod(dir, 0755)
```

Reply:
Existing code for handling a backup directory that has been zipped up. On the command line it is possible to restore by specifying an aforementioned zip archive.